### DISCUSSION TITLE
GitHub Au3 files and syntax highlighting

### DISCUSSION THREAD

**Discussion by mlipok** [genius257/au3pm#4] [Q&A]
Why is syntax highlighting for AutoIt files not accurate. E.g. there are no appropriately marked "string"

**Comment by genius257** [comment]
Can you provide an example?

**Reply by mlipok** [reply]
https://github.com/genius257/au3pm/blob/a0a95f8a7a52df1b95658763ac53c75c85450d9d/au3pm.au3#L367

![image](https://user-images.githubusercontent.com/11089482/106361026-f88a1d80-631b-11eb-83a8-139f7766e57a.png)

These following strings:
```autoit
Case "Int32", "Int64", "Double"
```
shouldn't be green ...? 

IMHO: At least they use the same color as $sJson but shouldn't.

**Reply by genius257** [reply]
They use slightly different colors that are to me not discernible with the human eye.
$sJson has color `#24292e` from the rule `color: var(--color-prettylights-syntax-storage-modifier-import)`
the strings has the color `#032f62` from the rule `color: var(--color-prettylights-syntax-string)`

If you enable GitHub Dark mode the color difference is better.
Else you need some browser extension for overriding the CSS vars.

**Reply by mlipok** [reply]
As for now I make request for help to github team.
And also searched for some reference documentation:
https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks#syntax-highlighting

EDIT:
And I also write here:
https://github.community/t/autoit-syntax-highlighting-issue/158673

**Reply by mlipok** [reply]
in a shortuct

When it comes to AutoIt syntax highlighting github.com  relate on this:
https://github.com/AutoIt/SublimeAutoItScript

Anybody who can help in fixing this could make a change and pool request to this repository.
